python/letsql/backends/let/hotfix.py

```diff
@@ -0,0 +1,165 @@
+import ibis.expr.operations as ops
+import ibis.expr.types.core
+import ibis.expr.types.relations
+from attr import (
+    field,
+    frozen,
+)
+from attr.validators import (
+    instance_of,
+)
+
+import letsql
+from letsql.expr.relations import (
+    CachedNode,
+    replace_cache_table,
+)
+from letsql.common.utils.hotfix_utils import (
+    maybe_hotfix,
+    none_tokenized,
+)
+
+
+@frozen
+class LETSQLAccessor:
+    expr = field(validator=instance_of(ibis.expr.types.core.Expr))
+    node_types = (ops.DatabaseTable, ops.SQLQueryResult)
+
+    @property
+    def op(self):
+        return self.expr.op()
+
+    @property
+    def cached_nodes(self):
+        return self.op.find(CachedNode)
+
+    @property
+    def storage(self):
```

Comment:
How would this work with multiple storages?

Reply:
`storage` works like `is_cached` and only refers to the top level expr/op being referenced; `storages` / `has_cached` refers to the entire graph of this expr